src/pages/cartpage/components/CartProductCard.css

```diff
@@ -0,0 +1,74 @@
+:root {
+    --white: white;
+    --black: black;
+    --hover-color: rgb(247, 242, 242);
+    --grey: grey;
+    --gainsboro: gainsboro;
+    --primary-blue: #2563EB;
+    --gold: #fdc434;
+}
```

Comment:
This will cause problems when there are variables with the same name. Only declare variables once in a separate main file and then use them in components instead of redeclaring for each component,